internal/events/handler.go

```diff
@@ -74,20 +74,22 @@ func (h *EventHandlerImpl) HandleEventBatch(ctx context.Context, batch EventBatc
 		}
 	}
 
-	changed, conf, statuses := h.cfg.Processor.Process(ctx)
+	changed, conf, g := h.cfg.Processor.Process(ctx)
```

Comment:
What's `g`?  Can we use a more descriptive variable name?

Reply:
Yep, missed this naming after testing.